tasks/configure.yml

```
@@ -1,13 +1,22 @@
 ---
+- name: copy amtool config
+  template:
+    force: true
+    src: "{{ alertmanager_amtool_config_file }}"
+    dest: "{{ alertmanager_amtool_config_dir }}/config.yml"
+    owner: alertmanager
+    group: alertmanager
+    mode: 0644
+
 - name: copy alertmanager config
   template:
     force: true
     src: "{{ alertmanager_config_file }}"
-    dest: "{{ alertmanager_config_dir }}/alertmanager.yml"
+    dest: "{{ _alertmanager_config_dir }}/alertmanager.yml"
```

```suggestion
    dest: "{{ alertmanager_config_dir }}/alertmanager.yml"
```
Why this is changed?

---

Not sure, I think it's fine now.